--- jquery-mobile.js	2014-04-08 15:15:02.000000000 +0200
+++ jquery-mobile-old.js	2014-04-08 15:16:17.000000000 +0200
@@ -19,6 +19,18 @@
 		});
 	} else {
 		// Browser globals
+        // kan.so patch
+        // Force jquery load, if not already done.
+        var jq = require("jquery");
+        if (root != window) {
+            // Enable loading from within kanso "load" function
+            root = window;
+        }
+        if (typeof root.jQuery == 'undefined') {
+            root.jQuery = jq;
+        }
+        // end kan.so patch
+
 		factory( root.jQuery, root, doc );
 	}
 }( this, document, function ( jQuery, window, document, undefined ) {
